scripts/generate_docs.sc

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env amm
+
+import java.time.Instant
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+
+/**
+  * Docs generation process:
+  *
+  * 1. copy docs/docs and docs/_layouts/docs.html part from 1.3, 1.4 and 1.5 branches to the temp dirs
```

Comment:
Hm. Why is this step necessary? Sorry for my ignorance.

Reply:
I don't want to mess with git branches too much, this step makes a clear separation between docs fetching from branches and following docs generation processes

Reply:
Ah. Thanks.